tests/e2e/backup_restore_suite_test.go

```diff
@@ -268,80 +264,79 @@ var _ = Describe("Backup and restore tests", func() {
 		func(brCase BackupRestoreCase, expectedErr error) {
 			runBackupAndRestore(brCase, expectedErr, updateLastBRcase, updateLastInstallTime)
 		},
-		Entry("MySQL application CSI", BackupRestoreCase{
+		Entry("MySQL application CSI", FlakeAttempts(3), BackupRestoreCase{
```

Comment:
nice, ginkgo already has this implemented

I would put number of trials as a variable, so when running locally, we can set to zero (and add docs about it)

Reply:
ah, and also add a comment about why we are retrying (link to issues for example)

Reply:
Yes. Found out that Gingko has such decorator :)

Ideally if we have flake attempts on not entire test runs but for example for backup/restore/compare separated, but don't want to make such radical change.

The comment is in the git commit log. I don't think comment here is right as we don't know if those are the only flakes here. We know 2 for sure that are already identified.

Reply:
@mateusoliveira43 added variable `flakeAttempts`